frontend/src/components/App/App.jsx

```diff
@@ -0,0 +1,25 @@
+import './App.css'
+import Navbar from '../Navbar/Navbar'
+import Search from '../Search/Search'
+import { useState } from 'react';
+
+export default function App() {
+  const [searchInput, setSearchInput] = useState("");
+  const [currentStock, setCurrentStock] = useState("");
+
+  function stockSearchSubmit() {
+    setCurrentStock(searchInput);
+  }
+
+  return (
+    <div className="app">
+      <Navbar />
+      <Search 
```

Comment:
I like that Search is a presentational (stateless) component, and App is the head. 
However, as you add features to the project, the code in App increases and includes a lot of non related features logic. 

A good way out of this is to add the state and logic relevant to searching to the Search component, and include a SearchView component if you want to keep a stateless one at the bottom.